core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -659,10 +660,10 @@ class GroupMetadataManager(val brokerId: Int,
       config.offsetsTopicNumPartitions
   }
 
-  private def getMessageFormatVersionAndTimestamp(partition: Int): (Byte, Long) = {
+  private def getMessageFormatVersionAndTimestampIfReplicaIsLocal(partition: Int): (Byte, Long) = {
```

Comment:
IMO, it would be better to indicate what the function is doing, if possible in its name itself. What name are you thinking?